rocketmq-streams-examples/src/main/java/org/apache/rocketmq/streams/examples/mutilconsumer/Producer.java

```diff
@@ -26,8 +26,7 @@
 import org.apache.rocketmq.remoting.common.RemotingHelper;
 import org.apache.rocketmq.streams.examples.rocketmqsource.ProducerFromFile;
 
-import static org.apache.rocketmq.streams.examples.rocketmqsource.Constant.NAMESRV_ADDRESS;
-import static org.apache.rocketmq.streams.examples.rocketmqsource.Constant.RMQ_TOPIC;
+import static org.apache.rocketmq.streams.examples.rocketmqsource.Constant.*;
```

Comment:
Not recommended using * aggregate all imports

Reply:
checkstyle not work ? you could set it in git workflow, when these checks all passed, we could merge it.

Reply:
resolved